matching-core/src/main/java/com/graphhopper/matching/util/HmmProbabilities.java

```diff
@@ -67,33 +55,12 @@ public double emissionLogProbability(double distance) {
      *                       consecutive map matching candidates.
      * @param linearDistance Linear distance [m] between two consecutive GPS
      *                       measurements.
-     * @param timeDiff       time difference [s] between two consecutive GPS
-     *                       measurements.
      */
-    public double transitionLogProbability(double routeLength, double linearDistance,
-                                           double timeDiff) {
-        if (timeDiff == 0) {
```

Comment:
With conflicting I mean e.g. when we calculate the correct fastest path going on a motorway and then still prefer the track besides the motorway which is shorter (but slower), because the matchings were slightly closer to the track. But using the fastest weighting for me implies that we should prefer matching to fast roads even if there are no time stamps attached. So more generally I think we should even use path.getWeight() so that one could in theory use a completely different road preference (not sure if this is useful, so maybe we should stick to time/fastest)